src/main/java/io/cdap/plugin/salesforce/parser/SalesforceQueryParser.java

```diff
@@ -16,6 +16,8 @@
 package io.cdap.plugin.salesforce.parser;
 
 import io.cdap.plugin.salesforce.SObjectDescriptor;
+
```

Comment:
why are we changing src/main/java? This should be a test only change

Reply:
Might have been done by mistake. Reverted it.